tests/pragma/jsfmt.spec.js

```diff
@@ -0,0 +1 @@
+run_spec(__dirname);
```

Comment:
can you explain this a little bit more please?

Reply:
See my comment below ⬇️